orbit-2-coroutines/src/main/java/com/babylon/orbit2/TransformFlow.kt

```diff
@@ -22,9 +22,17 @@ internal class TransformFlow<S : Any, E : Any, E2 : Any>(
     val block: suspend Context<S, E>.() -> Flow<E2>
 ) : Operator<S, E>
 
+/**
+ * The flow transformer flat maps incoming [Context] for every event into coroutine flows.
+ *
+ * The transformer executes on an `IO` dispatcher by default.
```

Comment:
should we be specific and just say Dispatchers.IO?

Reply:
I guess only advantage is this is agnostic of the framework

Reply:
We can link to it in the coroutine transformers but not the base one I guess